packages/ui-core/src/ButtonNew/ButtonNew.js

```diff
@@ -0,0 +1,43 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import PropTypes from "prop-types";
+import MaterialButton from "@material-ui/core/Button";
+
+import { ThemeProvider } from "@material-ui/styles";
+import { MaterialTheme } from "@hackoregon/ui-themes";
```

Comment:
Have you run numbers yet on how large `ui-core` is now? I'm curious how much weight material, emotion, and our ui themes adds.

Reply:
Could this be right?
<img width="788" alt="image" src="https://user-images.githubusercontent.com/7065695/84617761-d0c97080-ae84-11ea-8c07-bc4377e31fab.png">

Reply:
<img width="460" alt="image" src="https://user-images.githubusercontent.com/7065695/84618809-1c314e00-ae88-11ea-8fca-41cea8964385.png">

Reply:
This whole measuring things ad-hoc is a little problematic. I'd like to put in some automated checks for bundle size and performance.